webshooter.py

```diff
@@ -117,7 +119,7 @@ def jekyll_build(path):
     print(options_prompt)
     while static not in {"1", "2"}:
       try:
-        static = input("option> ").strip()
+        static = str(input("option> ")).strip()
```

Comment:
You should just use raw_input here; input calls `eval` on the string, so anything that isn't valid python barfs. Also, then it would already be a string.